examples/chip-tool-darwin/main.m

```diff
@@ -0,0 +1,30 @@
+/*
+ *   Copyright (c) 2021 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#import <CHIP/CHIPDevice.h>
+#import <CHIP/CHIPDeviceController.h>
+#import <CHIP/CHIPDevicePairingDelegate.h>
+#import <CHIP/CHIPError.h>
+#import <CHIP/CHIPManualSetupPayloadParser.h>
+#import <CHIP/CHIPPersistentStorageDelegate.h>
+#import <CHIP/CHIPQRCodeSetupPayloadParser.h>
+#import <CHIP/CHIPSetupPayload.h>
+#import <CHIP/gen/CHIPClustersObjc.h>
+#import <Foundation/Foundation.h>
+
+int main(int argc, const char * argv[]) { return EXIT_SUCCESS; }
```

Comment:
I'm confused.. this binary doesn't do anything?

Reply:
I guess the executable is to verify linking (i.e. no unresolved symbols)? Otherwise you can just build source_sets.

Reply:
> I guess the executable is to verify linking (i.e. no unresolved symbols)? Otherwise you can just build source_sets.

Yes. 2 of the last breakages were due to missing symbols.